code_samples/plugin_sample/resources/META-INF/plugin.xml

```diff
@@ -16,8 +16,8 @@
 
   <!-- http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/plugin_compatibility.html -->
   <!-- The unique identifiers of the plugins on which this plugin depends -->
-  <!-- com.intellij.modules.lang is included in every IntelliJ-based IDE, plugin will be available as well -->
-  <depends>com.intellij.modules.lang</depends>
+  <!-- com.intellij.modules.platform is included in every IntelliJ-based IDE, plugin will be available as well -->
```

Comment:
please do not change this. "lang" is available in all products

Reply:
Substitute "lang" for "platform" everywhere "platform" is used.